change/@azure-msal-browser-88c06603-4e47-4cc5-8d76-6aa3750c86de.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Fix pop format passed to STS in native flows (#6065)",
```

nit: Some of the linking doesn't work properly when wrapped in parens

```suggestion
  "comment": "Fix pop format passed to STS in native flows #6065",
```